server.go

```diff
@@ -81,9 +88,10 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 		}
 	}
 
-	if conf.Debug {
-		log.Level = logrus.DebugLevel
```

Comment:
looks like you deleted debug-level logging when you added the new config option?